frontend/src/concepts/pipelines/apiHooks/useExperiements.ts

```diff
@@ -1,10 +1,10 @@
 import * as React from 'react';
 import { usePipelinesAPI } from '~/concepts/pipelines/context';
-import useFetchState, { FetchStateCallbackPromise } from '~/utilities/useFetchState';
+import useFetchState, { FetchState, FetchStateCallbackPromise } from '~/utilities/useFetchState';
 import { ExperimentKF } from '~/concepts/pipelines/kfTypes';
 import { POLL_INTERVAL } from '~/utilities/const';
 
-const useExperiments = () => {
+const useExperiments = (): FetchState<ExperimentKF[], never[]> => {
```

Comment:
This is odd.

Discussed with @andrewballantyne and we should update the `useFetchState` generic types

Please remove the `Default` generic type from `useFetchSTate.ts`:
```
const useFetchState = <Type>(
```

This will cause some type errors in your current PR, that you need to address, where you've added explicit `FetchState` types with 2 generic params.